src/plugins/bubble/bubble.js

```diff
@@ -0,0 +1,177 @@
+					$element.css( {
+						bottom: ( $footer.outerHeight() - ( $document.outerHeight() - $window.outerHeight() - $window.scrollTop() ) + bottomY )
+					} );
+				} else {
+					$element.css( {
+						bottom: bottomY
+					} );
+				}
+			};
+
+			// Correct bubble positionning on load, on resize an on Y scroll if necessary
+			stickyUntilFooter( $selector );
+
+			$( window ).on( "resize scroll", function() {
+				stickyUntilFooter( $selector );
+			} );
+		} else {
+			$( "main" ).after( "<footer id=\"wb-info\" class=\"" + componentName + "-mrgn" + "\"></footer>" );
+		}
+
+		// Close notification aside bubble
+		$( ".notif-close", $selector ).on( "click", function( event ) {
+			event.preventDefault();
+			$( this ).parent().hide();
+			$selector.focus();
+
+			// Do not show notification until 7 days
+			var daySevenInMillionSeconds = 1000 * 60 * 60 * 24 * 7;
+			setNotificationStatusWithExpiry( componentName + "-notif", 1, daySevenInMillionSeconds );
+		} );
+	},
+
+
+	setNotificationStatusWithExpiry = function setNotificationStatusWithExpiry( key, value, ttl ) {
+		var now = new Date();
+
+		// `item` is an object which contains the original value
+		// as well as the time when it's supposed to expire
+		var item = {
+			value: value,
+			expiry: now.getTime() + ttl
+		};
+		localStorage.setItem( key, JSON.stringify( item ) );
+	},
+
+	getNotificationStatusWithExpiry = function getNotificationStatusWithExpiry( key ) {
+		var itemStr = localStorage.getItem( key );
+
+		// if the item doesn't exist, return null
+		if ( !itemStr ) {
+			return null;
+		}
+		var item = JSON.parse( itemStr );
+		var now = new Date();
+
+		// compare the expiry time of the item with the current time
+		if ( now.getTime() > item.expiry ) {
+
+			// If the item is expired, delete the item from storage
+			// and return null
+			localStorage.removeItem( key );
+			return null;
+		}
+		return item.value;
+	},
+
+
+/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+
+		// Start initialization
+		// returns DOM object = proceed with init
+		// returns undefined = do not proceed with init (e.g., already initialized)
+		var elm = wb.init( event, componentName, selector ),
+			$elm;
+		if ( elm ) {
+			$elm = $( elm );
+
+			var $bubbleElm = $( "<div class='" + componentName + "-wrap'></div>" );
+			$elm.wrap( $bubbleElm );
+
+			// Get JSON object to retrieve notification message
+			var data_wb_doaction_json = JSON.parse( $elm.attr( "data-wb-doaction" ) );
+			isNotif = getNotificationStatusWithExpiry( componentName + "-notif" );
+
+			var $notification = ( !isNotif ? "<p class=\"trans-left\">\r\n" +
+			"<span class=\"notif\">" + data_wb_doaction_json.notification + "</span>\r\n" +
+			"<a href=\"#\" class=\"notif-close\" title=\"Close chat notification\" aria-label=\"Close chat notification\" role=\"button\">×</a>" : "" );
```

Comment:
Remove the reference to the "chat"
Change anchor "<a>" for "<button>" and remove "role=button"